### PR TITLE
Security certifications fips copy updates

### PR DESCRIPTION
**Pull request by carkod**
## Done

Copy updates

## QA

- Check  https://ubuntu-com-10015.demos.haus/security/fips  against [copy docs](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4231

### CODE REVIEW

File: templates/security/fips.html [hunk 1]
```diff
@@ -29,20 +29,41 @@ <h2 class="p-heading--4">FIPS 140-2 validated cryptography for Linux workloads</
...
+      </p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Get NIST certified compliance</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        FIPS 140 ensures that cryptographic algorithms known to be secure are used for data protection, and they are thoroughly tested and attested by a laboratory accredited under the NIST’s Cryptographic and Security Testing (CST) <a class="p-link--external" href="https://www.nist.gov/nvlap/" />Laboratory Accreditation Program (LAP)</a> in the US and <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program">CCCS’s Cryptographic Module Validation Program (CMVP)</a> in Canada. <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide you with cryptographic packages that are tested and attested by atsec Information Security, a NIST accredited laboratory.
+      </p>
```
Comment (bethcollins92) [inline templates/security/fips.html:53]: You missed the two links at the bottom of this section - `Read more about the FIPS certification` and `contact us`

File: templates/security/fips.html [hunk 2]
```diff
@@ -165,45 +207,36 @@ <h2>Certified packages under FIPS 140-2</h2>
         </tr>
       </tbody>
     </table>
-    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140-2 validated modules</a>
+    <a class="p-button--positive js-invoke-modal" href="/security/certifications/docs/fips">Read more about the FIPS certification</a>
```
Comment (bethcollins92) [inline templates/security/fips.html:210]: I think this should be `p-button--neutral` judging by the copy doc as only the `Access the fips...` one is green

File: templates/security/fips.html [hunk 3]
```diff
@@ -165,45 +207,36 @@ <h2>Certified packages under FIPS 140-2</h2>
...
     <p>We recommend enabling the 'fips-updates' option that includes the security fixes. Our cloud images starting with 20.04 are pre-configured to enable security updates. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance phase</a>.</p>
   </div>
 </section>
 
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Free for personal use</h2>
+    <p>Canonical provides Ubuntu Advantage Essential subscriptions, which include FIPS, free of charge for individuals on up to 3 machines. For our community of Ubuntu members we will gladly increase that to 50 machines.</p>
+    <a class="p-button--positive" href="/advantage">Read more about the FIPS certification</a>
```
Comment (bethcollins92) [inline templates/security/fips.html:228]:
```suggestion
    <a class="p-button--positive" href="/advantage">Get a free subscription</a>
```

File: templates/security/fips.html [hunk 4]
```diff
@@ -165,45 +207,36 @@ <h2>Certified packages under FIPS 140-2</h2>
...
+    <p class="u-sv3">In September 2021 NIST is phasing out FIPS 140. Certifications under FIPS 140 remain valid until September 2026, and new products are expected to be certified <a class="p-link--external" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/fips-140-3-standards">under FIPS 140-3</a>. FIPS 140-3 is a combined effort of NIST and ISO with the Security and Testing requirements for cryptographic modules being published as ISO/IEC 19790 and ISO/IEC 24759. Canonical is preparing Ubuntu for the new certification, and intends to provide FIPS 140-3 certified cryptographic packages on a future LTS release of Ubuntu.</p>
   </div>
   <div class="u-fixed-width">
```
Comment (bethcollins92) [inline templates/security/fips.html:237]: you don't need this `<hr>` now you've deleted the last section

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good, just a few inline changes

File: templates/security/fips.html (hunk 1 above)
Comment (carkod) [reply templates/security/fips.html:53]: Done

File: templates/security/fips.html [hunk 5]
```diff
@@ -29,20 +29,45 @@ <h2 class="p-heading--4">FIPS 140-2 validated cryptography for Linux workloads</
...
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Get NIST certified compliance</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">
+        FIPS 140 ensures that cryptographic algorithms known to be secure are used for data protection, and they are thoroughly tested and attested by a laboratory accredited under the NIST’s Cryptographic and Security Testing (CST) <a class="p-link--external" href="https://www.nist.gov/nvlap/" />Laboratory Accreditation Program (LAP)</a> in the US and <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program">CCCS’s Cryptographic Module Validation Program (CMVP)</a> in Canada. <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide you with cryptographic packages that are tested and attested by atsec Information Security, a NIST accredited laboratory.
+      </p>
+    </article>
+  </div>
+  <p>
+    <a class="p-button--neutral" href="/security/certifications/docs/fips">Get Ubuntu with the FIPS140 validated modules</a>
```
Comment (bethcollins92) [inline templates/security/fips.html:57]: 1. These links should be in a `row` or `u-fixed-width` see below: 
Screenshot 2021-07-13 at 13 36 47
2. The text should say `Read more about the FIPS certification`

File: templates/security/fips.html [hunk 6]
```diff
@@ -73,33 +98,53 @@ <h2>Access FIPS images on the cloud</h2>
...
+        <script class="jsBrightTALKEmbedConfig" type="application/json">
+          { "channelId" : 6793, "language": "en-US", "commId" : 432536, "displayMode" : "standalone", "height" : "250" }
+        </script>
+        <script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
```
Comment (bethcollins92) [inline templates/security/fips.html:125]: I feel this should be a `p-strip` to keep the sections separate and then alternate the following ones

**Comment by spencerbygraves** [comment]
@carkod could you add a demo link please. Thanks.

**Comment by bethcollins92** [comment]
@spencerbygraves https://ubuntu-com-10015.demos.haus/security/fips here you go!

**Comment by spencerbygraves** [comment]
Thanks @carkod, a few things from me.

- Remove `is-bordered` from the second strip with 3 cards
- Align the buttons to the grid and update copy as mentioned by @bethcollins92
- Agree with @bethcollins92 that "Access FIPS images on the public cloud" could be a strip
- Are the next two sections related? If not they could also be strips

We can look at background colours after those changes.

Thanks

**Comment by carkod** [comment]
> ## Done
> Copy updates
> 
> ## QA
> * Check against [copy docs](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)
> 
> ## Issue / Card
> Fixes [canonical-web-and-design/web-squad#4231](https://github.com/canonical-web-and-design/web-squad/issues/4231)



> Thanks @carkod, a few things from me.
> 
> * Remove `is-bordered` from the second strip with 3 cards
> * Align the buttons to the grid and update copy as mentioned by @bethcollins92
> * Agree with @bethcollins92 that "Access FIPS images on the public cloud" could be a strip
> * Are the next two sections related? If not they could also be strips
> 
> We can look at background colours after those changes.
> 
> Thanks

Done. I modified all strips to do the alternating effect.

**Review by bethcollins92 — APPROVED** [review]
Code and QA LGTM but I think there should be a border or change in background colour between the hero and the first section

File: templates/security/fips.html [hunk 7]
```diff
@@ -10,10 +10,10 @@
...
-     <p><a class="p-button--positive js-invoke-modal" href="/support/contact-us">Get Ubuntu with the FIPS140–2 validated modules</a></p>
+     <h1>FIPS 140</h1>
+     <h2 class="p-heading--4">FIPS 140 validated cryptography for Linux workloads</h2>
+     <p>Developing and running workloads for U.S. government regulated and high security environments requires a long and expensive validation process. Reduce your accreditation timeline and pass on your validation costs with the FIPS 140 certified cryptographic packages of <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a>.</p>
+     <p><a class="p-button--positive js-invoke-modal" href="/security/certifications#get-in-touch">Get Ubuntu with the FIPS140 validated modules</a></p>
```
Comment (bethcollins92) [inline templates/security/fips.html:16]: Just realised this with Pete, the point of this `href` is so if someone has javascript disabled they will still be able to see a contact form - but this is linking to another modal which doesn't make sense. So I would link to `/security/contact-us`

**Comment by spencerbygraves** [comment]
@carkod thanks for updating. If you could just add `is-bordered` to the first strip I think we're done. Thanks.

File: templates/security/fips.html (hunk 7 above)
Comment (carkod) [reply templates/security/fips.html]: Done

**Comment by carkod** [comment]
> @carkod thanks for updating. If you could just add `is-bordered` to the first strip I think we're done. Thanks.

Done

**Comment by spencerbygraves** [comment]
@carkod the update doesn't show in the demo.

**Comment by carkod** [comment]
> @carkod the update doesn't show in the demo.

Sorry, it should be good now https://ubuntu-com-10015.demos.haus/security/fips

**Comment by spencerbygraves** [comment]
@carkod thanks

**Pull request merged**